### PR TITLE
Fix #771 & #772: Lowfi Landscape OptionsActivity & AppLanguageActivity

### LINKED ISSUE #771: Lowfi Landscape: OptionsActivity
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/7d442317-dd5a-40e7-adbd-25285d010874/OP-Options-Menu-

OptionsActivity should support landscape mode. Also add test cases for landscape mode too.

### LINKED ISSUE #772: Lowfi Landscape: AppLanguageActivity
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/2b8387b0-5ec6-4e42-bffa-c039128b2f12/OP-App-Language-/

`AppLanguageActivity` should support landscape mode. Also add test cases for landscape mode too.

This issue is currently blocked because the landscape mode mocks are not ready yet.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #772: Lowfi Landscape: AppLanguageActivity

Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/2b8387b0-5ec6-4e42-bffa-c039128b2f12/OP-App-Language-/

https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/7d442317-dd5a-40e7-adbd-25285d010874/OP-Options-Menu-/

This PR was already created by @MohamedMedhat1998 in #842 but he was stuck on CircleCI test cases failure again and again and therefore I have just recreated that PR.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/app_language_activity.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/story_list_app_bar_layout"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/app_language_activity.xml]: Please rename this app bar layout.

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt [hunk 2]
```diff
@@ -80,35 +80,36 @@ class AppVersionActivityTest {
...
-    onView(isRoot()).perform(orientationLandscape())
-    onView(
-      withId(
-        R.id.app_version_text_view
-      )
-    ).check(
-      matches(
-        withText(
-          String.format(context.resources.getString(R.string.app_version_name), BuildConfig.VERSION_NAME)
+  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly() {
```
Comment (veena14cs) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: Why App version test activity is in this PR.

**Review by veena14cs — CHANGES_REQUESTED** [review]
Please add configuration change testcases in OptionFragmentTest.

File: app/src/main/res/layout-land/app_language_activity.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/app_language_activity.xml]: Done

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt (hunk 2 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: Removed

**Comment by rt4914** [comment]
> Please add configuration change testcases in OptionFragmentTest.

I have updated the PR to finish landscape mode for `OptionsActivity` as well as `AppLanguageActivity` considering that all test cases are related to a single file.

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt [hunk 3]
```diff
@@ -80,36 +80,35 @@ class AppVersionActivityTest {
...
-      onView(isRoot()).perform(orientationLandscape())
-      onView(withId(R.id.app_version_text_view)).check(
-        matches(
-          withText(
-            String.format(
-              context.resources.getString(R.string.app_version_name),
-              BuildConfig.VERSION_NAME
-            )
-          )
+  fun testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly(){
```
Comment (veena14cs) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: This test case is failing.

**Review by veena14cs — APPROVED** [review]
LGTM.

**Pull request merged**